interface/patient_file/education.php

```diff
@@ -102,7 +102,7 @@
 
     <title><?php echo xlt('Education'); ?></title>
 
-    <?php Header::setupHeader('opener'); ?>
+    <?php Header::setupHeader(); ?> // Removed opener because this is not a dialog. sjp 12/14/17
```

Comment:
Isn't this called as a popup?

Reply:
(in tabs ui)
Confirmed. If you make an issue with an ICD10 code, then click on the issue in Issues screen, then this screen opens as a free popup. Also has the following outputted to the top:
// Removed opener because this is not a dialog. sjp 12/14/17
:)

Reply:
just a window to me that does not register with opener so had to treat as window for cancel. the other is just a boo boo..